src/components/shared/ContextMenu.css

```diff
@@ -123,13 +123,17 @@
 /* Use this wrapper for a content that needs to take the available space or if
  * it contains other HTML elements (eg <strong>) */
 .react-contextmenu-item-content {
-  flex: auto;
+  overflow: hidden;
+  min-width: 0;
+  flex: 1;
+  text-overflow: ellipsis;
 }
 
 /* Use a span with this class to add an icon to an item. */
 .react-contextmenu-icon {
   width: 16px;
   height: 16px;
+  flex-shrink: 0;
```

Comment:
optional: I usually use `flex: none` for this, but I don't mind, both are as explicit.

Reply:
Changed to `flex: none`.